playground/backend/cmd/server/server.go

```diff
@@ -107,8 +108,13 @@ func setupCache(ctx context.Context, appEnv environment.ApplicationEnvs) (cache.
+// "// Licensed to the Apache Software Foundation (ASF) under one or more\n// contributor license agreements.  See the NOTICE file distributed with\n// this work for additional information regarding copyright ownership.\n// The ASF licenses this file to You under the Apache License, Version 2.0\n// (the \"License\"); you may not use this file except in compliance with\n// the License.  You may obtain a copy of the License at\n//\n//    http://www.apache.org/licenses/LICENSE-2.0\n//\n// Unless required by applicable law or agreed to in writing, software\n// distributed under the License is distributed on an \"AS IS\" BASIS,\n// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n// See the License for the specific language governing permissions and\n// limitations under the License.\n\npackage beam_test\n\nimport (\n\t\"reflect\"\n\t\"testing\"\n\n\t\"github.com/apache/beam/sdks/v2/go/pkg/beam\"\n)\n\n// foolFn is a no-op CombineFn.\ntype foolFn struct {\n\tOutputType beam.EncodedType\n}\n\ntype foolAccum struct{}\n\nfunc (f *foolFn) CreateAccumulator() *foolAccum {\n\treturn &foolAccum{}\n}\n\nfunc (f *foolFn) AddInput(a *foolAccum, v beam.U) *foolAccum {\n\treturn a\n}\n\nfunc (f *foolFn) MergeAccumulators(a *foolAccum, b *foolAccum) *foolAccum {\n\treturn a\n}\n\nfunc (f *foolFn) ExtractOutput(a *foolAccum) beam.V {\n\treturn reflect.New(f.OutputType.T).Elem().Interface()\n}\n\nfunc TestCombineWithTypeDefinition(t *testing.T) {\n\t_, s := beam.NewPipelineWithRoot()\n\tin := beam.Create(s, 1, 2, 3)\n\tstrType := reflect.TypeOf(\"\")\n\tcombineFn := &foolFn{OutputType: beam.EncodedType{T: strType}}\n\toutput := beam.Combine(s, combineFn, in, beam.TypeDefinition{Var: beam.VType, T: strType})\n\tif output.Type().Type() != strType {\n\t\tt.Errorf(\"expect combine output type to be %v, got %v\", strType, output.Type().Type())\n\t}\n}\n"
```

Comment:
It seems that we do not need this one.

Reply:
Sorry it was draft version, needs it for testing